exporter/prometheusremotewriteexporter/README.md

```diff
@@ -61,6 +61,9 @@ The following settings can be optionally configured:
   - `enabled` (default = false): If `enabled` is `true`, a `_created` metric is
     exported for Summary, Histogram, and Monotonic Sum metric points if
     `StartTimeUnixNano` is set.
+- `max_batch_byte_size` (default = `3000000` -> `~2.861 mb`): Maximum size of a batch of
```

Comment:
nit: could we use `max_batch_size_bytes` instead? reference for similar property in an unrelated place https://prometheus.io/docs/prometheus/latest/feature_flags/#extra-scrape-metrics
`scrape_body_size_bytes`

Usually the suffix denotes the unit. 

another option is to use something like https://pkg.go.dev/github.com/docker/go-units#FromHumanSize

go-units is used in this project, but I think it is not worth it.

Otherwise I think this is a reasonable change and LGTM.

Reply:
@rapphil Sorry for the delay just refactored to `max_batch_size_bytes`

Reply:
Thank you!